repository-data/webfiles/src/main/resources/site/freemarker/intranet/document-list/task-list.ftl

```diff
@@ -0,0 +1,68 @@
+<#ftl output_format="HTML">
+
+<#include "../../include/imports.ftl">
+
+<#assign getRemoteSvg="uk.nhs.digital.freemarker.svg.SvgFromUrl"?new() />
+
+<@hst.setBundle basename="rb.generic.intranet.headers"/>
+
+<div class="nhsd-t-grid nhsd-t-grid--nested">
+    <div class="nhsd-t-row">
+        <div class="nhsd-t-col-12">
+            <#if wrappingDocument?? && wrappingDocument.title?? && wrappingDocument.title?has_content>
+                <#assign title>${wrappingDocument.title}</#assign>
+            <#else>
+                <#assign title><@fmt.message key="headers.tasks" /></#assign>
+            </#if>
+            <h2 class="nhsd-t-heading-m">${title}</h2>
+        </div>
+    </div>
+    <#if pageable?? && pageable.items?has_content>
+        <#list pageable.items as task>
+            <#if task?index % 2 == 0>
+                <div class="nhsd-t-row">
+            </#if>
+            <div class="hsd-t-col-12 nhsd-t-col-s-6 nhsd-!t-margin-bottom-6">
```

Comment:
Looks like you have a typo, should be, `nhsd-t-col-12`

Reply:
Well spotted.